type_test.go

```diff
@@ -238,6 +238,14 @@ var canonicalStringTests = []struct {
 		"symbols": ["a", "b"]
 	}]`,
 	out: `["int","string",{"name":"E","type":"enum","symbols":["a","b"]}]`,
+}, {
+	testName: "empty-record",
+	in: `{
+	"name": "R",
+	"type": "record",
+	"fields": []
+}`,
+	out: `{"name":"R","type":"record","fields":[]}`,
```

Comment:
Where is type `X` here?

Reply:
I'm sorry, I don't understand the question. What's the significance of type `X` here?

Reply:
According to the test https://github.com/heetch/avro/pull/86/files#diff-1b6d75ae831c3ba9a36a09615a8f68d5R42-R45 , there is a X field which is empty. shouldn't it appear in the Avro schema?

Reply:
That's a different test in a different package. Why do you think it would affect this one?

Reply:
I'm going to pass on by here. If there's actually an issue, I'll fix it in a later PR.